packages/plugin-conversation/test/integration/spec/share.js

```diff
@@ -217,6 +217,35 @@ describe(`plugin-conversation`, function() {
               });
           }));
       });
+
+      it(`shares a whiteboard`, () => spark.conversation.share(conversation, [{
+        file: sampleImageSmallOnePng,
+        actions: [{
+          type: `edit`,
+          mimeType: `application/x-cisco-spark-whiteboard`,
```

Comment:
That was part of the design put together by the team.  The mime type is support to specify the application that is used to open the file.   When a video is available for the whiteboard, we will use a video mimeType and a URL to the stream.  (Not all clients will consume that)